web/models/session.ex

```diff
@@ -1,5 +1,6 @@
 defmodule ElixirJobBoard.Session do
   alias ElixirJobBoard.User
+  require IEx
```

Comment:
I think we can remove this for now. Seems like it's similar to leaving a `require 'pry'` in a Rails app.